### PR TITLE
feat : added cart page 

### PR DESCRIPTION
**Pull request by Tanjum786**
Screenshot from 2022-02-17 02-43-26
  
**added cart page**

### CODE REVIEW

File: CSS/cart.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+.cart_card_container,
+.price_detailes_container {
+    margin-top: 3rem;
+    box-shadow: 2px 2px 2px 2px var(--boder-color);
+}
+
+.cart_container {
+    margin-top: 2rem;
+    margin-bottom: 2rem;
```
Comment (nishasen) [inline CSS/cart.css]: Use shorthand for this, margin : 2rem 0;

File: CSS/cart.css [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    width: 1.5rem;
+    border-radius: 50%;
+    height: 1.5rem;
+    border: 1px solid var(--gray-color);
+}
+
+.input_num {
+    width: 3rem;
+    padding: 0;
+    border: 1px solid black;
```
Comment (nishasen) [inline CSS/cart.css]: Use variable here

File: pages/cart.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>cart | E-Galaxy-store </title>
+    <link rel="stylesheet" href="https://css-ui-galaxy-c.netlify.app/CSS/base.css">
+    <link rel="stylesheet" href="/CSS/cart.css">
+    <script src="https://kit.fontawesome.com/8ddd15368f.js" crossorigin="anonymous"></script>
+
+</head>
+
```
Comment (nishasen) [inline pages/cart.html]: remove the extra space here

File: pages/cart.html [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+<body>
+    <div class="e_container">
+        <nav class="nav_bar">
+            <div>
+                <a href="/index.html">
+                    <h1 class="e_website_name">E-Galaxy-store</h1>
+                    <h5 class="ui_tagline">Find it, love it, buy it.</h5>
+                </a>
+
```
Comment (nishasen) [inline pages/cart.html]: extra space here

File: pages/cart.html [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+                        <button class="qunt_btn" id="qunt_decrease">-</button>
+                    </div>
+
+                    <div class="cart_btns">
+                        <button class="cart_remove_btn btn_cart">Remove Form Cart<a href="/"></a></button>
+                        <button class="cart_wishlist_btn btn_cart">Move to Wishlist<a href="/"></a></button>
+                    </div>
+                </div>
+            </div>
+
```
Comment (nishasen) [inline pages/cart.html]: extra space here

**Review by nishasen — COMMENTED** [review]
I have suggested some changes please go through those.

File: CSS/cart.css (hunk 1 above)
Comment (Tanjum786) [reply CSS/cart.css]: okay

**Review by nishasen — APPROVED** [review]
Looks good now, good work

**Review by vivekmehto — APPROVED** [review]
clean code looks good to me.

**Comment by Tanjum786** [comment]
> clean code looks good to me.

**Pull request merged**